src/i18n/locales/en-US.json

```diff
@@ -159,6 +159,14 @@
   "components.txhistory.txnavigationbuttons.receiveButton": "Receive",
   "components.txhistory.txnavigationbuttons.sendButton": "Send",
   "components.uikit.offlinebanner.offline": "You are offline. Please check settings on your device.",
+  "components.walletinit.balancecheck.balancecheckmodal.attention": "Attention",
+  "components.walletinit.balancecheck.balancecheckmodal.attentionDescription": "The balance check executed successfully, and we were able to match your wallet with the balance displayed below. Remember that the balance displayed should only match the one that you <strong>had on November 12th</strong>.",
```

Comment:
@v-almonacid it’s not displaying `html` maybe there is another way to do *bold*

Reply:
@nicarq yep, resolved in next PR